eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/LoginPrincipal.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.ucl.eidp.auth.jaas;
+
+import java.io.Serializable;
+import java.security.Principal;
+
+/**
```

Comment:
\[Checkstyle\] WARNING: First sentence of Javadoc is incomplete \(period is missing\) or not present\.